lib/spoom/coverage.rb

```diff
@@ -19,13 +19,24 @@ def snapshot(path: ".", rbi: true, sorbet_bin: nil)
 
         new_config = config.copy
         new_config.allowed_extensions.reject! { |ext| !rbi && ext == ".rbi" }
-
-        metrics = Spoom::Sorbet.srb_metrics(
+        flags = [
           "--no-config",
           "--no-error-sections",
           "--no-error-count",
           "--isolate-error-code=0",
           new_config.options_string,
+        ]
+
+        metrics = Spoom::Sorbet.srb_metrics(
+          *flags,
+          path: path,
+          capture_err: true,
+          sorbet_bin: sorbet_bin,
+        )
+        # Collect extra information using a different configuration
+        flags << "--ignore sorbet/rbi/gems sorbet/rbi/dsl"
+        secondary_metrics = Spoom::Sorbet.srb_metrics(
```

Comment:
What do you think about `metrics_without_rbis` instead?

Reply:
It's less accurate but slightly better. I was thinking this could be used further but for now `secondary` isn't informative enough.